content/pt-br/docs/tasks/configure-pod-container/pull-image-private-registry.md

```diff
@@ -0,0 +1,219 @@
+* `<your-name>` é o seu nome de usuário do Docker.
+* `<your-pword>` é a sua senha do Docker.
+* `<your-email>` é o seu email do Docker.
+
+Você definiu com sucesso as suas credenciais Docker no cluster como um Segredo chamado `regcred`.
+
+{{< note >}}
+Digitando segredos na linha de comando, pode armazená-los no histórico do seu shell, desprotegidos, e
+esses segredos também podem ser visíveis para outros usuários em seu PC durante o tempo que `kubectl` estiver em execução.
+{{< /note >}}
+
+
+## Inspecione o Segredo `regcred`
+
+Para entender o conteúdo do Segredo `regcred` que você criou, comece visualizando o Segredo em formato YAML:
+
+```shell
+kubectl get secret regcred --output=yaml
+```
+
+A saída é semelhante a esta:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  ...
+  name: regcred
+  ...
+data:
+  .dockerconfigjson: eyJodHRwczovL2luZGV4L ... J0QUl6RTIifX0=
+type: kubernetes.io/dockerconfigjson
+```
+
+O valor do campo `.dockerconfigjson` é uma representação em base64 das suas credenciais Docker.
+
+Para entender o que está no campo `.dockerconfigjson`, e converter os dados secretos em um
+formato legível:
+
+```shell
+kubectl get secret regcred --output="jsonpath={.data.\.dockerconfigjson}" | base64 --decode
+```
+
+A saída é semelhante a esta:
+
+```json
+{"auths":{"your.private.registry.example.com":{"username":"janedoe","password":"xxxxxxxxxxx","email":"jdoe@example.com","auth":"c3R...zE2"}}}
+```
+
+Para entender o que está no campo `auth`, converta o dado codificado em base64 para um formato legível:
+
+```shell
+echo "c3R...zE2" | base64 --decode
+```
+
+A saída, nome de usuário e senha concatenados com um `:`, é similar a esta:
+
+```none
+janedoe:xxxxxxxxxxx
+```
+
+Observe que os dados secretos contém o token de autorização semelhante ao seu arquivo local `~/.docker/config.json`.
+
+Você definiu com sucesso suas credenciais do Docker como um segredo chamado `regcred` no cluster.
+
+## Crie um Pod que use seu segredo
+
+Aqui está um manifesto para um exemplo de Pod que precisa acessar as suas credenciais do Docker em `regcred`:
+
+{{< codenew file="pods/private-reg-pod.yaml" >}}
+
+Baixe o arquivo acima no seu computador:
+
+```shell
+curl -L -O my-private-reg-pod.yaml https://k8s.io/examples/pods/private-reg-pod.yaml
+```
+
+No arquivo `my-private-reg-pod.yaml`, altere `<your-private-image>` com o caminho para uma imagem em um registro privado, como:
+
+```none
+your.private.registry.example.com/janedoe/jdoe-private:v1
+```
+
+Para puxar a imagem do registro privado, o Kubernetes precisa das credencias.
+O campo `imagePullSecrets` no arquivo de configuração especifica que
+o Kubernetes deve obter as credenciais de um segredo nomeado `regcred`.
+
+Crie um Pod que use o seu segredo e verifique se o pod está executando:
+
+```shell
+kubectl apply -f my-private-reg-pod.yaml
+kubectl get pod private-reg
+```
+
+## {{% heading "whatsnext" %}}
+
+* Aprenda mais sobre [Segredos](/docs/concepts/configuration/secret/)
```

Comment:
"Secrets" instead of "Segredos".

This page is currently translated to Portuguese and can be linked here.